OpenDreamClient/Interface/Controls/ControlInfo.cs

```diff
@@ -83,6 +83,7 @@ private void UpdateLabels() {
             _valueText.Clear();
 
             // Use the default color and font
+            SetTextColor(_owner.InfoDescriptor.TextColor.Value);
```

Comment:
Isn't this already covered by the `UpdateElementDescriptor()`? Was that not working?

Reply:
Did not work for me, no. I don't think its being called in the status tab.

Reply:
Think I found the issue: if the StatEntry is created after the color was set then it keeps the default black.

Calling SetTextColor here ends up running UpdateLabel twice so it would be better to instead set the `_textColor` in StatEntry's constructor.